src/components/Questionnaire/QuestionnaireEditor.tsx

```diff
@@ -1232,11 +1230,11 @@ function QuestionEditor({
             </div>
             <div className="flex gap-2 mt-1">
               <Badge variant="secondary">{type}</Badge>
-              {required && <Badge variant="secondary">Required</Badge>}
-              {repeats && <Badge variant="secondary">Repeatable</Badge>}
+              {required && <Badge variant="secondary">{t("required")}</Badge>}
+              {repeats && <Badge variant="secondary">{t("repeatable")}</Badge>}
               {type === "group" && questions && questions.length > 0 && (
                 <Badge variant="secondary">
-                  {questions.length} sub-questions
+                  {questions.length} {t("sub_questions")}
```

Comment:
use variables in translations to include the length in it. 

this may be incorrectly represented in other languages;

```
t("sub_questions_count", { count: questions.length })
```